### PR TITLE
[RyuJit] fix clang switch warning

### PR DESCRIPTION
**Pull request by sandreenko**
There was warning:
```
/opt/code/src/jit/stacklevelsetter.cpp:130:13: error: 121 enumeration values not handled in switch: 'GT_NONE', 'GT_LCL_VAR', 'GT_LCL_FLD'... [-Werror,-Wswitch]
    switch (node->OperGet())
            ^
```
add `default` to be explicit.

It is safe because we have the checker, that checks that all throw blocks are initialized (If there is a throw kind that is not covered by this switch its throw block won't be initialized and we hit assert in codegencommon about `acdStkLvlInit` when we generate jump to the throw helper block).

### CODE REVIEW

**Comment by sandreenko** [comment]
[Build error example.](https://ci.dot.net/job/dotnet_coreclr/job/master/job/x86_release_ubuntu/315/consoleText)

**Comment by sandreenko** [comment]
@dotnet-bot test Ubuntu x86 Debug	Queues Ubuntu x86 Debug Build
@dotnet-bot test Ubuntu x86 Release	Queues Ubuntu x86 Release Build

File: src/jit/stacklevelsetter.cpp [hunk 1]
```diff
@@ -151,6 +151,8 @@ void StackLevelSetter::SetThrowHelperBlocks(GenTree* node, BasicBlock* block)
             SetThrowHelperBlock(SCK_ARITH_EXCPN, block);
         }
         break;
+        default: // Other opers do not target throw blocks.
+            break;
```
Comment (echesakovMSFT) [inline src/jit/stacklevelsetter.cpp]: Is it correct to assume that we never gonna hit `default:` here?

Comment (sandreenko) [reply src/jit/stacklevelsetter.cpp]: No, we hit it and ignore them. All opers that do not throw will hit default case.

**Comment by sandreenko** [comment]
cc @dotnet/jit-contrib

**Comment by sandreenko** [comment]
@dotnet-bot test Ubuntu x86 Debug Queues Ubuntu x86 Debug Build
@dotnet-bot test Ubuntu x86 Release Queues Ubuntu x86 Release Build

File: src/jit/stacklevelsetter.cpp [hunk 2]
```diff
@@ -151,6 +153,10 @@ void StackLevelSetter::SetThrowHelperBlocks(GenTree* node, BasicBlock* block)
             SetThrowHelperBlock(SCK_ARITH_EXCPN, block);
         }
         break;
+        default:
+            // Other opers can target throw helper blocks only thought overflow exceptions.
```
Comment (BruceForstall) [inline src/jit/stacklevelsetter.cpp]: I don't understand this comment. Instead of "only thought" should it be "only due to"?

Comment (sandreenko) [reply src/jit/stacklevelsetter.cpp]: Yes, I will change this assert and comment soon. Thanks.

**Comment by sandreenko** [comment]
@dotnet-bot test Windows_NT x86 Checked r2r
@dotnet-bot test Windows_NT x86 Checked jitstress1
@dotnet-bot test Ubuntu x86 Debug Queues Ubuntu x86 Debug Build
@dotnet-bot test Ubuntu x86 Release Queues Ubuntu x86 Release Build

**Comment by sandreenko** [comment]
Ok, PR is ready for review;
the failure in Windows_NT x86 Checked R2R Build & Test is unrelated (issue #16710), others failures are infra problems.

**Pull request merged**